spec/models/assignment_spec.rb

```diff
@@ -2486,4 +2486,35 @@ def grouping_count(groupings)
       end
     end
   end
+
+  describe '#current_grader_data' do
+    let!(:assignment) { create :assignment }
+
+    context 'groupings with no members' do
+      let!(:grouping) { create :grouping, assignment: assignment }
+      it 'should not have sections' do
+        expect(assignment.current_grader_data[:groups][0][:_id]).to eq(grouping.id)
```

Comment:
@JunruL overall the tests are excellent. Just make sure to call `assignment.current_grader_data` once and save it in a variable. This is more efficient in general, and though it doesn't matter for these tests (since the data is so small), it's a good habit to adopt.